### DISCUSSION TITLE
Helm: Allow setting nodeselctor and tolerations on the init job

### DISCUSSION THREAD

**Discussion by jakedsouza** [apache/superset#18545] [Ideas]
**Is your feature request related to a problem? Please describe.**
All pods on our cluster require specific node selectors and tolerations to get scheduled. Pods without a nodeselctor and toleration remain in pending state and dont get deployed

**Describe the solution you'd like**
Id like to be able to configure these fields for the init job in a similar manner to the superset pods.

**Describe alternatives you've considered**

**Additional context**

**Comment by geido** [comment]
@jakedsouza I am not an expert in this field. I wish I could help you more. However, it looks like you know exactly how this would benefit you and probably other organizations with a similar configuration. Would you be interested in contributing to Superset and making it a reality? I would be happy to guide you through that process. 

@nytai might also have some ideas on the topic.

**Comment by jakedsouza** [comment]
Thanks @geido I sent out a PR for this https://github.com/apache/superset/pull/18063

**Reply by eschutho** [reply]
Hi @jakedsouza it looks like we need one more small version bump change in order to merge in your PR.
